FuelGaugeReader/FuelGaugeReader.ino

```diff
@@ -0,0 +1,38 @@
+#define USE_USBCON
+#include <ros.h>
+#include <std_msgs/Int64.h>
+#include <Arduino.h>
+std_msgs::Int64 fuel_level_percent_full;
+ros::Publisher pub_fuel_level("fuel_level_percent_full", &fuel_level_percent_full);
+ros::NodeHandle nh;
+
+const long LOOP_DELAY = 200;
+const int FUEL_GAUGE_READ_PIN = 61;
+const double VOLTAGE_ON_EMPTY = 2.45;
+const double VOLTAGE_ON_FULL = 3.49;
+const double GAUGE_INPUT_VOLTAGE = 5;
```

Comment:
I think the better word for this would be ANALOG_REF_VOLTAGE

Reply:
Fixed this. Thanks